src/main/java/com/google/sps/servlets/RecipeServlet.java

```diff
@@ -86,7 +88,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
   @Override
   public void doDelete(HttpServletRequest request, HttpServletResponse response)
-      throws IOException {}
+      throws IOException {
+    String recipeID = request.getParameter("recipeID");
+    if (recipeID == null) {
+      return;
```

Comment:
changed this in CommentServlet, but you might want to add a response.setStatus(HttpServletResponse.SC_BAD_REQUEST); or something like that